website/packages/docs/src/pages/css-extraction-webpack.mdx

```diff
@@ -1,49 +1,136 @@
 ---
 section: 50-Guides
-name: Webpack CSS Extraction
+name: Stylesheet extraction
 ---
 
-# Webpack CSS Extraction
+# Stylesheet Extraction
 
-Extracting to a static style sheet can be enabled with some extra configuration.
-We recommend only extracting when building an application for production.
+Stylesheet extraction is **highly recommended** for all component libraries and applications, so that generated class names can be combined and de-duplicated into one external stylesheet. This allows you to enjoy the decreased stylesheet size that comes with atomic styling.
 
-If you're building a design system or component library and shipping to NPM you shouldn't extract just yet,
-let your consumers do it in their app.
+There are two types of stylesheet extraction:
 
-## Setup
+- Platform stylesheet extraction
```

Comment:
Is this relevant for non Atlassians?

Reply:
I would say it is as if say you were a 2nd/3rd-party ecosystem partner using the Atlassian Design System and you are mixing `@atlaskit/button` (post-Compiled-world) with `@your/package` which distributes _runtime styles_ you will experience the same pain we are where now your `border: none` overrides `border-top: 1px solid blue` and your border-top disappears due to runtime injecting a `<style>` tag that results in this being loaded last.

…I think this can go away as a requirement (but technically not a blocker) if we fixed shorthand, @media, and pseudo specificity in other ways, but that's _months off_ even if we knew what that was and started it right away…right?